File: src/C#/Fibonacci/algorithm.cs
using System;

public class algorithm {
    public int cal(int number) {
        int num1 = 0, num2 = 1, temp;
        while(number >= 0) {
            temp = num2;
            num2 = num2 + num1;
            num1 = temp;
            number--;
        }
        return num1;
    }
}

File: src/Javascript/Fibonacci/algorithm.js
function algorithm(number) {
    let num1 = 0;
    let num2 = 1;
  
    while (number > 0) {
      const temp = num2;
      num2 = num2 + num1;
      num1 = temp;
      number--;
    }
  
    return num1;
}
  
console.log(algorithm(69));
// ^^ Output should be : 117669030460994